src/main/java/mate/academy/service/user/UserServiceImpl.java

```diff
@@ -24,6 +30,10 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
         }
 
         User user = userMapper.toEntity(requestDto);
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
+        Role userRole = roleRepository.findByName("USER")
+                .orElseThrow(() -> new RuntimeException("Role USER not found"));
```

Comment:
Let's use enum instead of USER in message
```suggestion
                .orElseThrow(() -> new EntityNotFoundException("Role USER not found"));
```